src/js/components/Select/doc.js

```diff
@@ -102,6 +102,9 @@ export const doc = Select => {
     multiple: PropTypes.bool.description(
       'Whether to allow multiple options to be selected.',
     ),
+    name: PropTypes.string.description(
+      `The name of the attribute when in a Form or FormField..`,
```

Comment:
hey, why two dots instead of one or three?

Reply:
nice catch! ❤️